packages/amplify_storage_s3/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.1.4
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/master/packages/amplify_storage_s3
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=1.10.0"
```

Comment:
Nit: This might be a good opportunity to rein in the consistency between `"` vs `'` usage in our pubspec. I think the preferred convention is `'` like you have it here for multi-part constraints and not even having quotes at all for single-part constraints i.e.:
```yaml
flutter: >=1.10.0
```

Reply:
Good idea.  

How did you get it to compile without ' ? as I get a compile error when I do that. 

![image](https://user-images.githubusercontent.com/11478656/118032153-2bc94900-b31c-11eb-8c33-3e7c25d90f76.png)

Reply:
Oh I think you'd want to use caret syntax there:
```yaml
flutter: ^0.10.0
```

Reply:
Nice switched to that.

Reply:
Cool. My bad